deepchem/utils/openvino_model.py

```diff
@@ -8,17 +8,62 @@
 
 import numpy as np
 
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
 import tensorflow as tf
 from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
 
 try:
   import mo_tf
-  from openvino.inference_engine import IECore, StatusCode, ExecutableNetwork
+  from openvino.inference_engine import IECore, StatusCode, ExecutableNetwork, IENetwork
   is_available = True
 except:
   is_available = False
 
 
+def softplus(x):
+  return torch.where(x < np.log(np.finfo(np.float32).max),
+                     torch.log(torch.exp(x) + 1), x)
+
+
+def pooling(graph, feat, weight=None, *, op='sum', ntype=None):
+  x = graph.nodes[ntype].data[feat]
+
+  if weight is not None:
+    x = x * graph.nodes[ntype].data[weight]
+
+  value = torch.reshape(x, (-1, graph.batch_num_nodes(ntype)[0], x.shape[1]))
+  return torch.mean(value, 1)
```

Comment:
Please move to `CGCNN_OV`

Reply:
We don't have `CGCNN_OV` any more. Should I restore it?